dynamiqs/time_array.py

```diff
@@ -494,7 +503,10 @@ def __add__(self, other: ArrayLike | TimeArray) -> TimeArray:
 
 
 class SummedTimeArray(TimeArray):
-    timearrays: list[TimeArray]
+    timearrays: list[TimeArray] = eqx.field(static=True)
```

Comment:
Is this required? This does not make all the underlying time arrays also static?

Reply:
Yes it's a vestige from the past, I shoud get rid of it